### PR TITLE
Raul dev

### PR DESCRIPTION
**Pull request by Raulp8**
start of bigquery implementation

### CODE REVIEW

File: shopsafe-backend/src/test/java/com/google/sps/BigQueryTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** */
```
Comment (glstewart17) [inline shopsafe-backend/src/test/java/com/google/sps/BigQueryTest.java]: Needs class definition comment

File: shopsafe-backend/src/main/java/com/google/sps/QueryNy.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.FieldValueList;
+import com.google.cloud.bigquery.Job;
+import com.google.cloud.bigquery.JobId;
+import com.google.cloud.bigquery.JobInfo;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.TableResult;
+import java.util.UUID;
+
+public class QueryNy {
```
Comment (glstewart17) [inline shopsafe-backend/src/main/java/com/google/sps/QueryNy.java]: Add class and function definition comments

File: shopsafe-backend/src/main/java/com/google/sps/QueryNy.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+        QueryJobConfiguration queryConfig = QueryJobConfiguration.newBuilder(
+                "SELECT confimed_cases FROM  'bigquery-public-data.covid19_nyt.us_counties' " +
+                        "WHERE lower(state_name) = '" + state.toLowerCase() + "' " +
+                        "AND lower(county) = '" + county.toLowerCase() + "' " +
+                        "ORDER BY date DESC"
+        ).setUseLegacySql(false).build();
+
+        JobId jobId = JobId.of(UUID.randomUUID().toString());
+        Job queryJob = bigquery.create(JobInfo.newBuilder(queryConfig).setJobId(jobId).build());
```
Comment (glstewart17) [inline shopsafe-backend/src/main/java/com/google/sps/QueryNy.java]: Could you explain what's going on here?

**Review by glstewart17 — APPROVED** [review]
Looks good aside from some needed comments

File: shopsafe-backend/src/main/java/com/google/sps/QueryNy.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+import com.google.cloud.bigquery.Job;
+import com.google.cloud.bigquery.JobId;
+import com.google.cloud.bigquery.JobInfo;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.TableResult;
+import java.util.UUID;
+
+public class QueryNy {
+
+    static public long exampleQuery(String state, String county) {
```
Comment (caroljli) [inline shopsafe-backend/src/main/java/com/google/sps/QueryNy.java]: Maybe add a method header/description here so we know what it returns given that there is an error.

**Review by caroljli — APPROVED** [review]

**Pull request merged**